### PR TITLE
Documentation Rendering Bug Fixes

### PR DESCRIPTION
**Pull request by jpjsonic**

### CODE REVIEW

File: mkdocs/topics/sources.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 Sources are data sources. They are databases that contain your data. This page describes some best practices how to set sources up. 
 
 !!! tip
-    There is specific documentation on how to connect to a variety of databases in the Configuration documentation ([ie. PostgresSQL](/../reference/configuration/targets/PostgresqlSource/))
+    There is specific documentation on how to connect to a variety of databases in the Configuration documentation ([ie. PostgresSQL](/../reference/configuration/sources/PostgresqlSource/))
```
Comment (tgsoverly) [inline mkdocs/topics/sources.md:5]: Heart

File: pyproject.toml [hunk 2]
```diff
@@ -11,7 +11,7 @@ visivo = "visivo.command_line:safe_visivo"
 
 [tool.poetry.dependencies]
 python = " >=3.10,<3.13"
-pydantic = ">=2.5.0"
+pydantic = ">=2.5.0, <=2.9.0"
```
Comment (tgsoverly) [inline pyproject.toml:14]: ?

File: visivo/parsers/schema_generator.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 
 def generate_schema():
-    schema = Project.model_json_schema()
+    schema = Project.model_json_schema(by_alias=False)
```
Comment (tgsoverly) [inline visivo/parsers/schema_generator.py:6]: Ohhhh

**Review by tgsoverly — APPROVED** [review]

File: pyproject.toml (hunk 2 above)
Comment (jpjsonic) [reply pyproject.toml:14]: It seems a little risky to let our pydantic dependency upgrade unbounded without validating that new versions of pydantic don't break Visivo

**Pull request merged**